code/src/Oldest.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import moment from 'moment'
+
+export const Oldest = ({ thoughts, onLikeClick }) => {
+  return (
+    <>
+      {thoughts.reverse().map((thought) => (
```

Comment:
Wow, so this component is pretty much the same as the "Newest" except for the reversed map?! Genious!!